py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/configuration.py

```diff
@@ -20,19 +20,24 @@ class Configuration:
     https_mbta_busloc_s3.s3.amazonaws.com_prod_VehiclePositions_enhanced.json.gz
     https_mbta_integration.mybluemix.net_vehicleCount.gz
     """
-    def __init__(self, filename: str) -> None:
+    def __init__(self,
+                 config_type: ConfigType=None,
+                 filename: str=None) -> None:
```

Comment:
i added this so that we can create a `Configuration` either from config type or filename.

if you prefer something else, i'm happy to switch it.